server/db/seeds.rb

```diff
@@ -0,0 +1,9 @@
+ApiUser.destroy_all
+
+ApiUser.create([
+                 {first_name: 'Mary', last_name: 'Black', city: 'Dublin', mentor: true, mentee: false, email: 'Test_email_1@email.com', password_digest: 'password1'}, 
```

Comment:
I'd suggest to use @example.com (reserved domain that apparently bounces emails but at least safe; some more info [here](https://stackoverflow.com/questions/1368163/is-there-a-standard-domain-for-testing-throwaway-email/1368309) ); ideally it should be an owned domain, but not sure if we have one.

Reason: if we send emails, they should not arrive to someone's inbox. If email is used for security purposes, then someone with access to this external inbox can gain access to the system - quite bad if admin access. Therefore, the above is a general recommendation.

Reply:
Cool didn't know about this, will use @example.com for now, we don't need to test anything with emails yet.